### DISCUSSION TITLE
Congrats to Paul and the future of mushroom

### DISCUSSION THREAD

**Discussion by ark-** [piitaya/lovelace-mushroom#716] [General]
With @piitaya joining NabuCasa (congrats man), does that mean the features of mushroom may slowly work their way into official HA frontend? Or do you think the whole project will merge?

Either way the future is very bright for BOTH projects.

**Comment by piitaya** [comment]
Thank you 🙂
It's not decided yet how Mushroom and Home Assistant Frontend will live together. There are lot of feedbacks on the current Mushroom cards. We will look the community feedbacks with the Home Assistant team to see how we can bring an awesome dashboard experience in Home Assistant 🤩

**Comment by antonio1475** [comment]
I was very pleasantly surprised when I just read the email and came here. Congratulations!! 🥳 

I'm very happy that every HA user will get a more Mushroom experience than the current default. Looking forward to see what's coming. Best of luck
